sympy/series/tests/test_limits.py

```diff
@@ -1295,3 +1295,11 @@ def test_issue_23752():
     assert limit(expr1, x, 0, '-') == -sqrt(3)*I
     assert limit(expr2, x, 0, '+') == sqrt(3)*I
     assert limit(expr2, x, 0, '-') == -sqrt(3)*I
+
+def test_issue_24276():
+    f=Lambda((x), log(tan(pi/2*tanh(x))).diff(x)) ; f(x)
```

Comment:
The `; f(x)` bit here doesn't seem necessary.

Reply:
And the parentheses around x are unnecessary.

Reply:
ok. 
should I delete this pull request and create a new one after making the changes.

Reply:
It is not necessary to make a new PR.